.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "org.alloytools.pardinus"]
```

Comment:
this is one of the major changes this PR applies, moving the (extended) Kodkod backend to an external repo and reference it as a git submodule; I don't know the consequences of this for the Alloy build process (I've been able to build Electrum with similar scripts just fine), but I'm open to alternative solutions to git submodules.